Assets/MRTK/Core/EventDatum/Teleport/TeleportEventData.cs

```diff
@@ -19,7 +19,7 @@ public class TeleportEventData : GenericBaseEventData
         /// <summary>
         /// The teleport hot spot.
         /// </summary>
-        public IMixedRealityTeleportHotSpot HotSpot { get; private set; }
+        public IMixedRealityTeleportHotspot Hotspot { get; private set; }
```

Comment:
Just a reminder that we should include a description of this breaking change in the release notes.